### DISCUSSION TITLE
Lspsaga hoverdoc

### DISCUSSION THREAD

**Discussion by stonewall1901** [nvim-treesitter/nvim-treesitter#4213] [General]
I am using Lspsaga.nvim plugin, The code'keymap("n", "K", "cmd>Lspsaga hover docCR>") used to work properly, but now when I press "K" it says "[Lpsaga.nvim] please install markdown parser in nvim-treesitter"

Can you give me some advice on how to fix this? please explain in details.

**Comment by clason** [comment]
Did you install the markdown parser(s)?

**Comment by stonewall1901** [comment]
nope, I do not know how to install markdown. please explain.

**Comment by clason** [comment]
Please read the documentation.
